structure_tests/structure_test.go

```diff
@@ -36,48 +40,50 @@ type FileContentTest struct {
 }
 
 type StructureTest struct {
-	Commands           []CommandTest       `json:"commands"`
-	FileExistenceTests []FileExistenceTest `json:"file_existence"`
-	FileContentTests   []FileContentTest   `json:"file_contents"`
+	CommandTests       []CommandTest
+	FileExistenceTests []FileExistenceTest
+	FileContentTests   []FileContentTest
 }
 
 func TestRunCommand(t *testing.T) {
-	for _, tt := range tests.Commands {
+	for _, tt := range tests.CommandTests {
 		t.Logf("COMMAND TEST: %s", tt.Name)
 		var cmd *exec.Cmd
 		if tt.Flags != "" {
 			cmd = exec.Command(tt.Command, tt.Flags)
 		} else {
 			cmd = exec.Command(tt.Command)
 		}
+		t.Logf("Executing: %s", cmd.Args)
 		var outbuf, errbuf bytes.Buffer
 
 		cmd.Stdout = &outbuf
 		cmd.Stderr = &errbuf
 
-		err := cmd.Run()
+		if err := cmd.Run(); err != nil {
+			// The test might be designed to run a command that exits with an error.
```

Comment:
Yeah I played around with a few different ways, but I think adding a field with the exit code is the best way. We can track that separately.